### ISSUE TITLE
Accessibility (major): Landing page - ensure that the reading order of content is logical [argo-003]

### ISSUE THREAD

**Issue by aaron-collier** [sul-dlss/argo#4161]
## Description

The programmatic (DOM) order does not match the visual reading order.

In DOM-order, the "Limit your search" facet region comes after "Search results" region, but appears before the "Search results" in visual L-R reading order.

Moreover, the "Search results" instructions direct the user to "select a facet on the left to begin" but this facet region comes *after* the "Search results" in DOM-order (i.e., the screen reader user must navigate to the *right* to find the "Limit your search" region).

## Suggestion

- [x] Put the `<section id=""sidebar">` before the `<section id=""content">` in DOM-order.
- [x] Rewrite the "Welcome to Argo!" instructions to avoid using visual/spatial references. For example, instead of: "Enter one or more search terms or select a facet on the left to begin", consider something like "To begin, enter one or more search terms, or select a facet under the 'Limit your search' panel."
- [ ] Consider moving the "Welcome to Argo!" message outside of the Search results content container (see issue ID argo-001, Level 1 Hierarchical Order).

https://docs.google.com/spreadsheets/d/1m3obE89_mBCZpRkWAjJAXEr2ZfrUirzjvsJWqYH-G6Y/edit#gid=0

**Comment by edsu** [comment]
@aaron-collier will confer with @mjgiarlo about this work's connection with upstream blacklight.
